.mocharc.yml

```diff
@@ -0,0 +1 @@
+reporter: spec
```

Comment:
Since this is the [default](https://mochajs.org/api/mocha#reporter), mocha config is not necessary.